policies/sql/queries/bigquery/datasets_publicly_accessible.sql

```diff
@@ -0,0 +1,31 @@
+-- SELECT
+--     d.project_id,
+--     d.id,
+--     d.friendly_name,
+--     d.self_link AS dataset_link,
+--     a.special_group AS "group",
+--     a."role"
+-- FROM gcp_bigquery_datasets d
+--     JOIN gcp_bigquery_dataset_accesses a ON
+--         d.id = a.dataset_id
+-- WHERE a."role" = 'allUsers'
+--     OR a."role" = 'allAuthenticatedUsers';
+
+
+INSERT INTO gcp_policy_results (resource_id, execution_time, framework, check_id, title, project_id, status)
+SELECT DISTINCT d.id                                                                                   AS resource_id,
+                :'execution_time'::timestamp                                                           AS execution_time,
+                :'framework'                                                                           AS framework,
+                :'check_id'                                                                            AS check_id,
```

Comment:
Should framework and check_id be set to their real values in all the queries, or will this be done later?

Reply:
Ah I see, cool, thanks